src/newrelic.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['tta-smart-hub-development'],

```

Comment:
Does this need to match the `NEW_RELIC_APP_NAME` value in the manifest?  I'm asking because `tta-smarthub-((env))` will compile to `tta-smarthub-dev` not `tta-smart-hub-development`.

Reply:
This setting is for local environments. `NEW_RELIC_APP_NAME` will override this setting, I'll add a comment to clarify that.